Slider/Slider.js

```diff
@@ -267,60 +89,271 @@ const SliderBase = kind({
 				handleDecrement
 			])
 		),
+
 		onKeyUp: handle(
 			forProp('disabled', false),
 			forwardWithPrevent('onKeyUp'),
 			forProp('activateOnSelect', true),
 			forKey('enter'),
 			forward('onActivate')
 		)
-	},
-
-	computed: {
-		className: ({activateOnSelect, active, showAnchor, styler}) => styler.append({
-			activateOnSelect,
-			active,
-			showAnchor
-		}),
-		knobStep: validateSteppedOnce(props => props.knobStep, {
-			component: 'Slider',
-			stepName: 'knobStep',
-			valueName: 'max'
-		}),
-		step: validateSteppedOnce(props => props.step, {
-			component: 'Slider',
-			valueName: 'max'
-		}),
-		tooltip: ({tooltip}) => tooltip === true ? ProgressBarTooltip : tooltip
-	},
-
-	render: ({css, disabled, focused, tooltip, ...rest}) => {
-		delete rest.activateOnSelect;
-		delete rest.active;
-		delete rest.onActivate;
-		delete rest.knobStep;
-		delete rest.showAnchor;
-
-		return (
-			<UiSlider
-				{...rest}
-				aria-disabled={disabled}
-				css={css}
-				disabled={disabled}
-				progressBarComponent={
-					<ProgressBar css={css} />
-				}
-				tooltipComponent={
-					<ComponentOverride
-						component={tooltip}
-						css={css}
-						visible={focused}
-					/>
-				}
-			/>
-		);
+	}, props, spotlightAccelerator);
+
+
+	// if the props includes a css map, merge them together
+	let mergedCss = componentCss;
+	if (css) {
+		const allowedClassNames = Object.keys(componentCss);
+		mergedCss = mergeClassNameMaps(componentCss, css, allowedClassNames);
 	}
-});
+
+	const componentClassName = classnames(
+		componentCss.slider,
+		className,
+		{
+			[componentCss.active]: active,
```

Comment:
I think you missed `activateOnSelect` here.

Reply:
Please see https://github.com/enactjs/sandstone/pull/851#discussion_r601915938
I intended to remove it because it is not used anymore.

Reply:
👌